server/auto_updater/auto_updater_runner.py

```diff
@@ -0,0 +1,87 @@
+from dataset_updater import (
+    gather_new_papers,
+    generate_vector_counts,
+    merge_files,
+    update_dictionaries,
+)
+
+from embedding_bin_updater import (
+    generate_centroid_dataset,
+    generate_SAUCIE_coordinates,
+    update_paper_bins_stats,
+)
+
+from gensim.models import Word2Vec
+
+if __name__ == "__main__":
+    # New temp paper files
+    temp_dir_filename = "local_data/new_papers_added_dir.tsv"
+    temp_embed_filename = "local_data/new_papers_embedding.tsv"
+    temp_token_count_filename = "local_data/new_paper_token_count.tsv"
+
+    # Current paper files
+    current_pmc_dir_filename = "local_data/pmc_oa_file_list.tsv"
+    paper_dataset_filename = "local_data/paper_dataset_full.tsv.xz"
+    paper_landscape_file = "local_data/paper_dataset_tsne_square.tsv"
```

Comment:
Can't find this file in the repo either. Can you add it either to the repo or to the Google cloud bucket?